modules/openapi-generator/src/main/resources/typescript-angular/api.service.mustache

```diff
@@ -300,7 +300,13 @@ export class {{classname}} {
             })
         {{/isCollectionFormatMulti}}
         {{^isCollectionFormatMulti}}
-            {{#useHttpClient}}formParams = {{/useHttpClient}}formParams.append('{{baseName}}', {{paramName}}.join(COLLECTION_FORMATS['{{collectionFormat}}'])){{#useHttpClient}} as any || formParams{{/useHttpClient}};
+            if (useForm) {
+                {{paramName}}.forEach((element) => {
+                    {{#useHttpClient}}formParams = {{/useHttpClient}}formParams.append('{{baseName}}', <any>element){{#useHttpClient}} as any || formParams{{/useHttpClient}};
+            })
```

Comment:
@topce is this line (306) indented correctly? Should it have 4 more additional spaces?

Reply:
Hi @wing328 @macjohnny 
Thank you for reviewing code
@wing328  Probably you are right 
Code formatting is easy 
But template formatting is more difficult ;-)
Please fell free to fix this if you think it is appropriate 
Luckily for us we are not using white space programming language ;-)